tardis/plasma/standard_plasmas.py

```diff
@@ -73,7 +73,12 @@ def assemble_plasma(config, model, atom_data=None):
             raise ValueError('No atom_data option found in the configuration.')
 
         logger.info('Reading Atomic Data from %s', atom_data_fname)
-        atom_data = atomic.AtomData.from_hdf(atom_data_fname)
+
+        try:
+            atom_data = atomic.AtomData.from_hdf(atom_data_fname)
+        except:
```

Comment:
you should never have blanket exceptions. Always catch specific exceptions - you want your code to fail if it hits an unknown problem.

Reply:
@wkerzendorf okay, noted. I've made this change in the next update.